fortio/stage/Makefile

```diff
@@ -0,0 +1,53 @@
+
+# TODO: parametrize the parent directory yaml etc instead of copy pasta
+# on the other hand it is safer to not accidentally damage the prod version.
+
+deploy-fortio:
+	$(ISTIO_DIR)/bin/istioctl kube-inject --debug=false -f fortio.yaml | kubectl apply -f -
+
+all: cert-setup ingress-setup cert-issue deploy-fortio
+
+PROJECT:=istio-io
+MACHINE_TYPE:=n1-standard-1
+NUM_NODES:=3
+EXTRA_CLUSTER_OPTIONS:=--no-enable-legacy-authorization --enable-autoupgrade --enable-autorepair --enable-autoscaling --min-nodes "3" --max-nodes "10"
+create-cluster:
+	gcloud container clusters create fortio-stage --zone us-west1-c --project $(PROJECT) --machine-type=$(MACHINE_TYPE) --num-nodes=$(NUM_NODES) $(EXTRA_CLUSTER_OPTIONS)
+
+ISTIO_VERSION:=0.5.0
+ISTIO_DIR:=istio-$(ISTIO_VERSION)
+install-istio:
+	curl -L https://git.io/getLatestIstio | ISTIO_VERSION=$(ISTIO_VERSION) sh -
```

Comment:
btw that part (and line 6) is a fairly good smoke test of our release - cc @hklai